src/snowflake/snowpark/_internal/compiler/large_query_breakdown.py

```diff
@@ -63,6 +64,12 @@
 _logger = logging.getLogger(__name__)
 
 
+class SkipLargeQueryBreakdownCategory(Enum):
+    ACTIVE_TRANSACTION = "active transaction"
```

Comment:
@sfc-gh-aalam let's add a new telemetry_constant files for compiler component, i can see we will have many telemetry added for the compilation stage later